packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.css

```
@@ -21,16 +22,24 @@
 
 .optionsList {
   margin: 0;
-  padding: 0;
+  padding: var(--space-small);
   list-style: none;
 }
 
 .option {
-  padding: var(--space-small) var(--space-base);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: var(--space-small);
+  color: var(--color-blue);
```

Can we use the semantic value here? 
```suggestion
  color: var(--color-heading);
```